### PR TITLE
LIWO-496: Add printing to JPG/PNG in print function

### PR DESCRIPTION
**Pull request by sjoerdbeentjes**

### CODE REVIEW

File: src/components/LiwoMap.vue [hunk 1]
```diff
@@ -68,6 +64,34 @@ export default {
...
+        map.setZoom(zoom)
+      }
+
+      if (center) {
+        const [lat, lng] = center.split(',')
+        map.setView([lat, lng])
+      }
+    },
+    addPositionListeners (map) {
+      map.on('move', debounce((e) => {
```
Comment (SiggyF) [inline src/components/LiwoMap.vue:83]: looks good

File: src/components/MapImageControl.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            <h3 class="control-label">Locatie</h3>
+
+            <p class="map-image-control__location">Coördinaten linksboven: {{ origin.join(', ') }}</p>
+            <p>Zoomniveau: {{ zoomLevel }}</p>
+          </div>
+
+          <button class="btn primary" @click.prevent.stop="exportAsImage">Exporteren als afbeelding</button>
+        </form>
+
+        <!-- TODO: add coordinates of top left corner -->
```
Comment (SiggyF) [inline src/components/MapImageControl.vue]: Did you plan to add this?

File: src/map.config.js [hunk 3]
```diff
@@ -95,7 +95,7 @@ async function getServices () {
   }
   let url = 'config/webconfig.json'
   // TODO: how shall we configure this? Discuss with Peter
-  // url = 'config/webconfig-rws.json'
+  url = 'config/webconfig-rws.json'
```
Comment (SiggyF) [inline src/map.config.js]: can you comment this back? We now have a public test url which should be available by default.

**Review by SiggyF — APPROVED** [review]
Looks good, some minor comments.

File: src/components/MapImageControl.vue (hunk 2 above)
Comment (sjoerdbeentjes) [reply src/components/MapImageControl.vue]: Forgot to remove the comment, the functionality is in there

**Pull request merged**